EIPS/eip-4635.md

```diff
@@ -0,0 +1,312 @@
+---
+eip: 4635
+title: Semi-fungible token standard
+author: Bo Lin (@linvictor88), Danyang Li (@muren102), Frank Shen (@d22shen)
+discussions-to: https://ethereum-magicians.org/t/eip-4635-semi-fungible-token-standard/7923
+status: Draft
+type: Standards Track
+category: ERC
+created: 2021-12-16
+requires: 721
+---
+
+## Simple Summary
```

Comment:
This field has been deprecated in favor of the `description` preamble field. Please update accordingly.

Reply:
OK. thanks